### ISSUE TITLE
flypast

### ISSUE THREAD

**Issue by emuelecnoob** [libretro/flycast#1043]
Hi

I'm new here, can anyone help me with how to run Atomiswave/Naomi rom on flycast? I put all the bios in the bios folder and roms in Atomiswave or Naomi folder. But I got at error message. I can run Dreamcast roms. I dont know what I did wrong.

Regards
Emuelecnoob

**Comment by bslenul** [comment]
For Atomiswave/Naomi you need to use MAME roms. For the BIOS, in your `retroarch/system/dc/` folder you need "awbios.zip" (Atomiswave BIOS) and "naomi.zip" (Naomi BIOS), make sure they're in a "dc" subfolder or it won't work!

But this place is for reporting bugs, if you need more help, please joind the Discord: https://discordapp.com/invite/27Xxm2h
or use the subreddit: https://www.reddit.com/r/RetroArch/
or the forums: https://forums.libretro.com/

**Comment by emuelecnoob** [comment]
> For Atomiswave/Naomi you need to use MAME roms. For the BIOS, in your `retroarch/system/dc/` folder you need "awbios.zip" (Atomiswave BIOS) and "naomi.zip" (Naomi BIOS), make sure they're in a "dc" subfolder or it won't work!
> 
> But this place is for reporting bugs, if you need more help, please joind the Discord: https://discordapp.com/invite/27Xxm2h
> or use the subreddit: https://www.reddit.com/r/RetroArch/
> or the forums: https://forums.libretro.com/

Hi Bslenul,

Thanks for your advice, I don’t get what you mean by use Mame roms... Isn’t Mame roms for Mame emulators?

**Comment by bslenul** [comment]
Yes they're for MAME, but Flycast can use them: https://docs.libretro.com/library/flycast/#how-to-play-naomi-games (same for Atomiswave).

I think the core can also load other type of Naomi/Atomiswave roms, in .lst and .bin format or something, but I never bothered with this format as I've always read that MAME roms were recommended.

**Comment by emuelecnoob** [comment]
Is the rom sets same as Naomi/ atomiswave? I have the lst/ bin files but it doesn’t work on flycast.
